src/components/Navbar.js

```diff
@@ -0,0 +1,48 @@
+import { NavLink } from 'react-router-dom';
+import logo from '../logo/planet.png';
+import './Navbar.css';
+
+function NavBar() {
+  return (
+    <nav className="navbar">
+      {/* <Link "> */}
```

Comment:
[OPTIONAL] Please ensure that any unused commented codes are removed from the file. Thank you.